public/app/views/shared/_digital.html.erb

```diff
@@ -8,8 +8,7 @@
       } %>
       <% if representative_link_to_digital_materials?(record) %>
         <%
-          num_instances = record['json']['instances'].length
-          digitized_link_text = t('digital_object._public.browse_number_digital_objects_in_collection', num: num_instances)
+          digitized_link_text = t('digital_object._public.browse_number_digital_objects_in_collection', num: @n_digital_objects)
```

Comment:
This reintroduces direct use of an instance var while others were pulled out recently. To align with those recent changes (assuming that's desirable) add a local `n_digital_objects: @n_digital_objects`: https://github.com/archivesspace/archivesspace/blob/master/public/app/views/resources/show.html.erb#L24

Reply:
This has been resolved in the latest force push.

Reply:
@mark-cooper I asked about the recent instance variable changes that you refer to above [in that PR](https://github.com/archivesspace/archivesspace/pull/2934#discussion_r1109050975). I had a hunch there was a convention for using instance variables, just curious for more info.

Reply:
@brianzelip this is one of those "there's no right or wrong way" exactly, although my sense is that Rails best practices are to prefer locals over instance vars (haven't read the guides recently though). The PUI does a lot of both, but I generally favor locals because then partials become like "template functions" with arguments etc. that leads to better structured views in my experience.

Couple of articles from a quick Google search:

https://andycroll.com/ruby/only-use-locals-in-view-partials/
https://www.visuality.pl/posts/should-i-use-instance-variables-in-rails-views

Rails docs:
https://guides.rubyonrails.org/action_view_overview.html